src/pngwriter.cc

```diff
@@ -2023,6 +2023,11 @@ void pngwriter::write_png(void)
 ///////////////////////////////////////////
 void pngwriter::plot_text( char * face_path, int fontsize, int x_start, int y_start, double angle, char * text, double red, double green, double blue)
```

Comment:
you are already making it ABI and API incompatible with the other changes, it is going into `0.6.0` and will be at least compatible on the API with the `std::string` constructor.